legal-api/src/legal_api/utils/datetime.py

```diff
@@ -30,5 +34,5 @@ def utcnow(cls):
 
     @classmethod
     def from_date(cls, date_obj):
-        """Get a datetime object from a date object."""
-        return datetime(date_obj.year, date_obj.month, date_obj.day)
+        """Get a datetime object from a date object and localize it to Pacific time."""
+        return datetime(date_obj.year, date_obj.month, date_obj.day, tzinfo=PACIFIC_TZ)
```

Comment:
As I don't know where else `from_date` is used, this seems dangerous.

Generally, the API should work with UTC dates.

The AGM date is an exception because it's working with "Pacific dates".

I'd be tempted to clone this method, name the copy `from_date_pacific`, and then use that only where you need it.

Reply:
It uses UTC dates. I'm using LegislationDatetime now. There are some tests to check it.